api/LMPlatform.UI/Controllers/AdministrationController.cs

```diff
@@ -60,7 +60,7 @@ public DataTablesResult<StudentViewModel> GetCollectionStudents(DataTablesParam
             catch (Exception e)
             {
                 return DataTableExtensions.GetResults(
-                    new List<StudentViewModel> {new StudentViewModel {Login = e.StackTrace}}, dataTableParam, 1);
+                    new List<StudentViewModel> {new StudentViewModel {UserName = e.StackTrace}}, dataTableParam, 1);
```

Comment:
Hmm, Why we use UserName to store error stack trace? Do we use this code?

Reply:
No, this method is never used by front-end side. There is no request for this.